js/ViewportResizer.js

```diff
@@ -0,0 +1,223 @@
+/* eslint-disable no-unused-vars */
```

Comment:
I think the resizer w/h inputs/display should be below the example since it is a secondary action and in other design tools that information is located in this region. 
Also, the corner control for resizing should be polished up a bit to look like the attached. Can you revisit it to make it look like this?
![updated-resize](https://cloud.githubusercontent.com/assets/102225/24375179/65b51f92-1305-11e7-96ea-a733b9abd674.png)